test/OpenTelemetry.Tests/Implementation/Trace/Propagation/TraceContextActivityTest.cs

```diff
@@ -59,9 +59,7 @@ public void TraceContextFormatCanParseExampleFromSpec()
             Assert.Equal(ActivityTraceId.CreateFromString(TraceId.AsSpan()), ctx.TraceId);
             Assert.Equal(ActivitySpanId.CreateFromString(SpanId.AsSpan()), ctx.SpanId);
 
-            // TODO: when ActivityContext supports IsRemote
-            // Assert.True(ctx.IsRemote);
-
+            Assert.False(ctx.IsRemote);
```

Comment:
On the left it says Assert.True but the new version has Assert.False. Should Extract be setting IsRemote = true?

Reply:
That's a good question! would that be in the spec? or is that something from us to decide?

Reply:
I think it's safe to just make it true. `IsRemote` means the ActivityContext came from an upstream source. Extract is building the ActivityContext from the upstream source. So an extract context really implies IsRemote=true. At least in my thinking 😄 

IsRemote = false would only be for spans created in-process, using something like ActivitySource.StartActivity.

Actually, the spec does help in this case! From [SpanContext](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/api.md#spancontext):
> IsRemote is a boolean which is true if the SpanContext was propagated from a remote parent. When extracting a SpanContext through the Propagators API, its IsRemote flag MUST be set to true, whereas the SpanContext of any child spans MUST have it set to false.

Reply:
If using propagator.Extract - then it should be isRemote=true. The ActivityContext.IsRemote will be true for the extracted context. When an activity is created with this extracted context as its parent, then newly created activities' ActivityContext will NOT have isRemote=true.

Reply:
@cijothomas / @CodeBlanch , thanks for the comments. Just pushed another change. Let me know if that's right!